src/typescript/sdk/src/indexer-v2/types/index.ts

```diff
@@ -642,6 +643,11 @@ export const GuidGetters = {
     eventName: EVENT_NAMES.State,
     guid: `${formatEmojis(data)}::${EVENT_NAMES.State}::${getMarketNonce(data)}` as const,
   }),
+  candlestick: ({ market_id, start_time, period }: DatabaseJsonType["normal_candlesticks"]) => ({
+    // Not a real contract event, but used to classify the type of data.
+    eventName: ARENA_CANDLESTICK_NAME,
+    guid: `${ARENA_CANDLESTICK_NAME}::${market_id}::${period}::${start_time}`,
```

Comment:
```suggestion
    eventName: NORMAL_CANDLESTICK_NAME,
    guid: `${NORMAL_CANDLESTICK_NAME}::${market_id}::${period}::${start_time}`,
```

these `guid`s are just to avoid duplicate insertions in the global state, but if they conflict (arena with normal ones) it might lead to some weird issues if a melee id ever matches a market id